src/app/debug/table/table.component.ts

```diff
@@ -183,6 +191,7 @@ export class TableComponent implements OnInit, OnDestroy {
         next: (value: Report[]) => {
           this.setUniqueOptions(value);
           this.tableSettings.reportMetadata = value;
+          this.tableDataSource.data = [...value];
```

Comment:
Would this not work? Using the spread operator loops over the list which is unecessary if the resulting list doesn't contain anything other than the list
```suggestion
          this.tableDataSource.data = value;
```

Reply:
Added the change. It would work with or without. I was more afraid of shared mutable state and that the material table would change the order of the original values. Turns out it does not.